database/src/main/java/invivible/database/models/objects/Rating.java

```diff
@@ -32,7 +33,11 @@
 
   @Id
   private Long id;
+  private Long userId;
+  @Transient
   private User user;
+  private Long poiId;
+  @Transient
   private PointOfInterest poi;
```

Comment:
warum haben wir hier nochmal den Poi, wenn in der Entity auch die poiId drin ist?

Reply:
Ja brauchen wir vllt gar nicht.